lib/DDG/Spice/Justdeleteme.pm

```diff
@@ -0,0 +1,33 @@
+package DDG::Spice::Justdeleteme;
+
+# ABSTRACT: Search JusteDelte.Me for the provider
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
```

Comment:
Naturally all the boilerplate will be removed before final PR, but I initially pushed this here to get help on the layout issue. Still having that problem.